docs/src/pages/components/dialog.mdx

```diff
@@ -0,0 +1,373 @@
+
+```typescript
+import Button from '@pluralsight/ps-design-system-button'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import { P } from '@pluralsight/ps-design-system-text'
+import React from 'react'
+
+function Example() {
+  return <div className="example-grid--col-2">
+    <Dialog disableFocusOnMount><ExampleContent /></Dialog>
+    <Dialog onClose={_ => alert('Closing')} disableFocusOnMount><ExampleContent /></Dialog>
+  </div>
+}
+
+export default Example
+
+const ExampleContent = _ => (
+  <div style={{ maxWidth: '300px' }}>
+    <P>
+      Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+    </P>
+    <div style={{ textAlign: 'right' }}>
+      <Button>Primary button</Button>
+    </div>
+  </div>
+)
+```
+
+### Modal
+
+A modal Dialog will be shown fullscreen with an overlay behind and positioned in the center of the viewport.
+
+When a modal Dialog (or non-modal Dialog) are shown on the screen, it will take browser focus. In the case of a modal Dialog, the rest of the body of the application will continue to be visible through the translucent overlay. But it should be hidden. To hide it from screenreaders, add an `aria-hidden=true` attribute to your application's containing node, and make sure Dialog is mounted outside of that node. To hide it from keyboard users, add a `tabindex="-1"` to that same application node.
+
+```typescript
+import Button from '@pluralsight/ps-design-system-button'
+import { colorsBackgroundLight } from '@pluralsight/ps-design-system-core'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import { Heading } from '@pluralsight/ps-design-system-text'
+import React from 'react'
+
+function Example() {
+  const [isOpen, setOpen] = React.useState(false)
+  return <>
+    <div className="app" aria-hidden={isOpen}>
+      <Button onClick={() => setOpen(!isOpen)}>Open Modal</Button>
+    </div>
+    {isOpen && 
+      <Dialog modal onClose={() => setOpen(false)} aria-label="example dialog" style={{ zIndex: 1 }}>
+        <Heading>
+          <h2>Lorem ipsum dolor sit amet.</h2>
+        </Heading>
+        <div
+          style={{
+            background: colorsBackgroundLight[2],
+            height: '176px'
+          }}
+        />
+        <div className="buttons">
+          <Button appearance={Button.appearances.stroke}>Secondary</Button>
+          <Button>Primary button</Button>
+        </div>
+      </Dialog>
+    }
+  </>
+}
+
+export default Example
+```
+
+## Guidelines
+
+In most cases, dialogs can assume the user to digest the content following a Z-Pattern. Readers will start in the top/left, move horizontally to the top/right and then diagonally to the bottom/left before finishing with another horizontal movement to the bottom/right. By that measure, place your actions at the bottom/right, with the primary action on the right.
+      
+<Guideline
+  do={
+    <Dialog disableFocusOnMount>
+      <Text.Heading>
+        <h3>Title</h3>
```

Comment:
all the `h3` tags are getting the `.h3` style from the mdx components and it's giving them a bunch of extra margin. no idea why